pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1422,6 +1423,23 @@ func TestNodeShouldRunDaemonPod(t *testing.T) {
 						ObjectMeta: metav1.ObjectMeta{
 							Labels: simpleDaemonSetLabel,
 						},
+						Spec: resourcePodSpec("", "50M", "0.5"),
+					},
+				},
+			},
+			nodeCondition:         []v1.NodeCondition{{Type: v1.NodeOutOfDisk, Status: v1.ConditionTrue}},
+			wantToRun:             true,
+			shouldSchedule:        false,
+			shouldContinueRunning: true,
+		},
+		{
+			ds: &extensions.DaemonSet{
```

Comment:
This is func func TestNodeShouldRunDaemonPod(t *testing.T) {(t *testing.T) 
Should its name be changed?
The test case is TestNodeShouldNotRunDaemonPod(t *testing.T)